### ISSUE TITLE
Note text not showing up in interface

### ISSUE THREAD

**Issue by monotasker** [glushchenko/fsnotes#1407]
### Description

I installed fsnotes and set my notes directory to a folder with many plain text documents already created. When I open the FSNotes interface the list of note titles appears to be empty, but searching does bring up a list of notes. It's just that their titles aren't being displayed. Once I click on a result its title appears in that results list row. 

The text of the selected note often isn't visible at all until I click in the note editor pane. Then (sometimes) the note text will appear.

This is all on MacOS 12.6

### To Reproduce

1. Set default storage to my folder of existing .txt files
2. Set default format to .txt
3. Search in notes UI
4. See apparently empty results list
5. Click on result item
6. Title for that item appears in its result list row, Note text remains invisible
7. Click in note editor pane
8. Note text (sometimes) becomes visible

### Expected behavior

I expect the results list to be populated with the filenames of my existing note files.
I also expect the text of the existing note to appear in the note editor pane as soon as I select a note from the results list

### FSNotes version

5.4.5

### macOS/iOS version

12.6

**Comment by glushchenko** [comment]
I cannot reproduce this behavior. Perhaps you will have some ideas?

**Comment by monotasker** [comment]
Ah, I solved it! The folder was in Dropbox and I had it set to "online-only" with Dropbox. The problem was solved by setting the whole folder to be "available offline." This isn't a problem in my case, but I wonder whether there's a way for FSNotes to read titles from the online-only folder (the same way they can be displayed by Finder) and open the text of each file on demand?

**Comment by glushchenko** [comment]
All you need is to have the files physically on your hard drive, how you will synchronize them is not up to FSNotes

**Comment by monotasker** [comment]
Right. I'm just wondering if others will run into the same issue, because the default now is for Dropbox to only download the actual file contents on demand. Other apps seem to be able to see the filenames of these files (just like they appear in Finder as if they were on the local drive), and seem to be able to download them on demand. I think Dropbox's integration is supposed to be transparent that way. But I don't know what the mechanics of that are.

In other words, I don't think you should be worrying about how people sync their files. But Dropbox is a common enough storage tool that if there's a hiccup with reading from Dropbox folders that might be a usability problem for some, or at least cause confusion. It did in my case, and almost made me give up on the app. 

Thankfully I didn't, because it's a fantastic app. Thank you for your great work.
